From 2375306dfb1e89a4129e4d0e513dd21a8200f0bb Mon Sep 17 00:00:00 2001
From: Aubrey Li <aubrey.li@intel.com>
Date: Sun, 11 May 2014 06:05:21 -0500
Subject: [PATCH 093/152] Add power button irq debug message

This message is for debug purpose only. If the power button on T100
cannot wake the system up from freeze or suspend, we want to see if
the driver is properly configured and the interrupt could occur

Signed-off-by: Aubrey Li <aubrey.li@intel.com>
---
 drivers/pinctrl/pinctrl-baytrail.c |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/pinctrl/pinctrl-baytrail.c b/drivers/pinctrl/pinctrl-baytrail.c
index e599834..d1c6541 100644
--- a/drivers/pinctrl/pinctrl-baytrail.c
+++ b/drivers/pinctrl/pinctrl-baytrail.c
@@ -34,6 +34,7 @@
 #include <linux/io.h>
 #include <linux/pm_runtime.h>
 #include <linux/pinctrl/pinctrl.h>
+#include <linux/suspend.h>
 
 /* memory mapped register offsets */
 #define BYT_CONF0_REG		0x000
@@ -327,6 +328,8 @@ static void byt_gpio_irq_handler(unsigned irq, struct irq_desc *desc)
 	unsigned virq;
 	int looplimit = 0;
 
+	pm_suspend_dbg("baytrail pinctrl irq occurs, num %d\n", irq);
+
 	/* check from GPIO controller which pin triggered the interrupt */
 	for (base = 0; base < vg->chip.ngpio; base += 32) {
 
-- 
1.7.10.4

